docs/api.rst

```diff
@@ -6,7 +6,7 @@
     Acts
     ----
 
-    .. automodule:: cobalt.act
+    .. automodule:: cobalt
```

Comment:
should this now read cobalt.hierarchical?

Reply:
We may need to rethink the documentation structure. In `api.rst` we should have a top-level entry for each of the document types, and then `automodule` the appropriate classes within it.